app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -46,6 +53,15 @@ class HomeFragment : Fragment() {
         // todo: saved instance state?
         initTiles()
         initUrlInputView()
+
+        settingsButton.alpha = 0.4f
```

Comment:
nit: you can block related code operating on one object with `with()`:
```kotlin
with (settingsButton) {
    alpha = 0.4f
    setOnFocusChangeListener { ...
    ...
}
```

It's not always more readable but could be helpful. It's especially awesome for things like:
```kotlin
override fun onViewCreated...
    ...
    initSettingsButton()
}

fun initSettingsButton() = with (settingsButton) {
    alpha = 0.4f
    ...
}
```

Which I might recommend here. I find it's these initializers that create the broken windows `[1]` since they quickly sprawl out of control because everyone adds a few lines at a time and suddenly it's multiple pages long and impossible to understand.

`[1]:` Reference to broken window theory